src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1586,5 +1586,24 @@ private void ResetScrollIndicator(bool forced = false)
 			}
 		}
 		#endregion
+
+#if __CROSSRUNTIME__ || __MACOS__
+		private static bool _warnedAboutZoomedContentAlignment;
+
+		[NotImplemented]
+		private void UpdateZoomedContentAlignment()
+		{
+			if (_warnedAboutZoomedContentAlignment)
+			{
+				return;
+			}
+
+			_warnedAboutZoomedContentAlignment = true;
+			if (this.Log().IsEnabled(LogLevel.Warning))
+			{
+				this.Log().LogWarning("Zoom-based content alignment is not implemented on this platform.");
```

Comment:
@MartinZikmund we should be using: 
https://github.com/unoplatform/uno/blob/6437cd6fe66a06ea007f9cf7477aa1a13dd620e3/src/Uno.Foundation/Metadata/ApiInformation.cs#L125

to align the log level.